services/worker-manager/src/providers/provider.js

```diff
@@ -47,7 +47,7 @@ class Provider {
   async prepare() {
   }
 
-  async provision({workerPool, existingCapacity}) {
+  async provision({workerPool, workerInfo}) {
```

Comment:
There's some docs for this in `services/worker-manager/docs/providers.md` that should be updated, too.

Also, the parent class serves as a kind of documentation:
https://github.com/taskcluster/taskcluster/blob/3d4c26818fbdf5b2322f2a1286146a359f0b76d9/services/worker-manager/src/providers/provider.js#L50-L51